web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
     flex-flow: column nowrap;
+    margin: 0;
+    padding: 0;
+    border: solid 1px var(--color-border-dropdown-menu);
+    background-color: var(--color-background-dropdown-menu);
+    max-height: 85vh;
+    min-width: 230px;
+    overflow-x: hidden;
+    user-select: none;
+    border-radius: 6px;
+    box-shadow: var(--box-shadow-navbar-dropdown-menu);
```

Comment:
`box-shadow`, too, is overridden by `#gear-menu-dropdown`, although perhaps this variable ought to be called from there?

Reply:
This seems like a good default value for future navbar popovers (like help menu) and is used by personal menu right now.